.github/workflows/taskrunner.yml

```diff
@@ -24,10 +24,16 @@ jobs:
       uses: actions/setup-python@v3
       with:
         python-version: "3.8"
-    - name: Install dependencies
+    - name: Install dependencies ubuntu
+      if: matrix.os == 'ubuntu-latest'
```

Comment:
If we are on latest `ubuntu` or `windows` versions, we may want to have `python-version: "3.9"` or  even `python-version: "3.10" as our default test matrix environment.

Also we need to add a workflow for Python 3.11.